test/Libraries/WorkflowTests/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="DynamoVisualProgramming.LibG_225" version="2.1.0" targetFramework="net47" />
```

Comment:
I'd imagine there will be 3 of these references?

Reply:
Yes, still generating those ...